lib/engine/step/g_1828/buy_special.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require_relative '../buy_special'
+
+module Engine
+  module Step
+    module G1828
+      class BuySpecial < BuySpecial
+        def can_buy_special?(entity)
+          @game.can_buy_coal_marker?(entity)
+        end
+
+        def short_description
+          'Coal Marker'
+        end
+
+        def items
+          [{ description: 'Coal Marker', cost: 120 }]
```

Comment:
should we make a real item class?

Reply:
or should we instead use assign?

Reply:
Sure, I can make it a class.

I looked at assign. If I understand it correctly, the player clicks on the object to assign and then on what to assign it to. The coal marker doesn't really have the first half of that, so in my opinion, a button makes sense.